.buildkite/scripts/release/k8s-manifests.sh

```diff
@@ -14,14 +14,21 @@ set -eu
 
 ROOT="$(cd "$(dirname "$0")"; pwd)/../../.."
 
-retry() { "$ROOT/hack/retry.sh" 5 "$@"; }
+source "$ROOT/.buildkite/scripts/common/trigger.sh"
+source "$ROOT/.buildkite/scripts/common/operator-image.sh"
 
-get_image_tag() {
-  buildkite-agent meta-data get operator-image --default "" | cut -d':' -f2
-}
+retry() { "$ROOT/hack/retry.sh" 5 "$@"; }
 
 main() {
-  local version=${BUILDKITE_TAG:-$(get_image_tag)}
+  echo "--- generate-manifests"
+
+  # set IMAGE_NAME and IMAGE_TAG before generating manifests
+  operator::set_image_vars "${TRIGGER:-$(trigger::set_from_env)}"
```

Comment:
I think it's better to get the image names and tags using the two scripts than reading the operator image from meta-data.